html/index.html

```diff
@@ -0,0 +1,58 @@
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="description" content="This is a web site for newbie amateur astronomers." />
+    <title>What Can I See Tonight?</title>
+
+    <link rel="stylesheet" href="../css/reset.css" type="text/css" />
+    <link rel="stylesheet" href="../css/styles.css" type="text/css" />
+    <link href='http://fonts.googleapis.com/css?family=Roboto:400,400italic,700' rel='stylesheet' type='text/css'>
+  </head>
+  <body>
+      <div class="container clearfix">
+    <header>
+      <div class="container clearfix">
+            <h1>What Can I See Tonight?</h1>
+      </div>
+      <div class="container-menu clearfix">
+            <ul>
+              <li><a href="index.html">Main</a></li>
+              <li><a href="blog.html">Blog</a></li>
+              <li><a href="resources.html">Resources</a></li>
+              <li><a href="about.html">About</a></li>
+            </ul>
+      </div>
+
+    </header>
+
+    <main>
+        <form>
+                <input type="text" id="location" placeholder="Enter your location">
+                <input type="submit" value="Update" id="submit-btn">
+        </form>
+
+        <form>
+                <input type="text" id="date" placeholder="Enter today's date">
+                <input type="submit" value="Update" id="submit-btn">
+        </form>
+    </main>
+<!--The form inputs are for geographical location and date (both of those could default to the present values (i.e. where the user actually is at that time,) if that's possible. The user could go with those, or put in different values (i.e. if they were planning to go to another location on another day). After the inputs are made, the page will display weather conditions, moon phase/rise time, and events such as meteor showers depending on what (free) APIs are available-->
```

Comment:
you can request the user's location from the browser: https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/Using_geolocation